src/styles/_fonts.scss

```diff
@@ -19,11 +19,3 @@ $font-size-mobile-title-small: 12px;
 $font-size-mobile-body: 14px;
 $font-size-mobile-title-small: 12px;
 $font-size-mobile-body-small: 10px;
-
-* {
-  font-size: $font-size-mobile-body;
-
-  @include media-breakpoint-up(lg) {
-    font-size: $font-size-desktop-body;
-  }
-}
```

Comment:
@Skafec , just to let you know that I removed this. There is an issue with rich text editor because it has custom font size on h1, h2, h3.. elements. In rich text editor, when you set "Title" as a heading, it will look like this:

```
<h1>Title</h1>
```

The font size for this is now 2.5rem (lets say).
Now, when you want to color it, it will do this:

```
<h1><span color="blabla">Title</span></h1>
```
Because of our global rules, span will override the font height to 14px and we don't want that.

I've passed through the application and haven't seen any issues after removing these global font size rules. If you find some glitches, just to let you know that this might break it.